README.md

```diff
@@ -24,10 +52,30 @@ TRELLO_DONE_LIST_ID
 
 Install geckodriver and Firefox as we need these two for running selenium tests.
 
-Once the setup script has completed and all packages have been installed, start the Flask app by running:
-```bash
-$ flask run
+
+## Running within Docker
+
+### Building docker image
+To build the docker image run the following command
+
 ```
+docker build --target development --tag todo-app:dev .
+docker build --target production --tag todo-app:prod .
+```
+
+### Running the container
+
+To run the production container as a daemon run following command
+```
+docker run -p 8000:5000 --env-file .env -d todo-app:prod
```

Comment:
Any reason you chose to publish it on the host's port 8000 instead? Might catch someone out who just copies this command and expects it to run on port 5000 like the dev one.